package.json

```diff
@@ -14,11 +14,14 @@
   "author": "Blackbaud",
   "license": "MIT",
   "dependencies": {
-    "@blackbaud/skyux": "2.0.0-beta.26",
-    "@blackbaud/stache": "2.0.0-beta.0"
+    "@blackbaud/skyux": "2.0.0-beta.29",
+    "@blackbaud/stache": "2.0.0-beta.0",
+    "@blackbaud/skyux-builder-plugin-stache-code-block": "1.0.0",
```

Comment:
Plugins shouldn't be included in the template package.json, only in Stache proper. They *should* be listed in skyuxconfig.json, however.